CHANGES.md

```diff
@@ -1,5 +1,12 @@
 Change Log
 ==========
+
+### 1.39 - 2017-11-01
+
+* Breaking Changes
+  *
+* Added the ability to load Cesium's assets from the local file system [#5830](https://github.com/AnalyticalGraphicsInc/cesium/issues/5830)
```

Comment:
This description should probably be more specific as Cesium still can't access the local file system in typical scenarios because of browser security.  Does this new feature support only Chromium with security disabled?  Or does it impact other browsers as well?  It's unclear to me is `xhr.status === 0` is common or not.

Reply:
To answer my own question, status 0 for file protocol is very common and even jQuery deals with it: https://github.com/jquery/jquery/blob/master/src/ajax/xhr.js#L17